mysql-test/mroonga/storage/index/primary/varchar/t/in_select_empty_string.test

```diff
@@ -25,7 +25,7 @@ DROP TABLE IF EXISTS ids;
 CREATE TABLE ids (
   id varchar(7) PRIMARY KEY,
   parent_id varchar(7)
-) DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
+) ENGINE=Mroonga DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
```

Comment:
This is needless. Because we use Mroonga as the default storage engine in test.

Reply:
I see.
Thank you for your comment.